### PR TITLE
no closing brace was provided

### PR DESCRIPTION
**Pull request by vladyslavstartsev**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Slevomat Coding Standard
+ignoreAssignmentsInsideFunctionCalls# Slevomat Coding Standard
```
Comment (kukulich) [inline README.md:1]: @vladyslavstartsev Typo?

Comment (vladyslavstartsev) [reply README.md:1]: definitely

Comment (vladyslavstartsev) [reply README.md]: @kukulich done

**Comment by kukulich** [comment]
Thanks.

**Pull request merged**